### PR TITLE
Readme.md file updated

### PR DESCRIPTION
**Pull request by RedonAdemi**

### CODE REVIEW

File: ReviveIT/Application/Common/Exceptions/UrlGuard.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+namespace Application.Common.Exceptions
+{
+    public static class UrlGuard
+    {
+        public class UnsafeUrlException : Exception
+        {
+            public UnsafeUrlException(string message) : base(message) { }
+        }
+        // List of unsafe patterns or keywords, including single characters
```
Comment (donatgosalcii) [inline ReviveIT/Application/Common/Exceptions/UrlGuard.cs:15]: Remove comments on this class for a cleaner code.

File: ReviveIT/WebUI/Middlewares/ExceptionMiddleware.cs [hunk 2]
```diff
@@ -3,63 +3,94 @@
 
 namespace WebUI.MiddleWares
 {
+    // Middleware class for handling exceptions and unsafe URLs
```
Comment (donatgosalcii) [inline ReviveIT/WebUI/Middlewares/ExceptionMiddleware.cs:6]: Remove comments here too.

**Review by donatgosalcii — CHANGES_REQUESTED** [review]
Check this PR, you should close this and commit the readme file to the other PR, because it will just overwrite the code of the PR that gets approved first.

**Comment by RedonAdemi** [comment]
> Check this PR, you should close this and commit the readme file to the other PR, because it will just overwrite the code of the PR gets approved first.

Its seperated from other pr

**Pull request closed without merging**